config/routes.rb

```diff
@@ -104,14 +104,17 @@
     resources :operations, only: [:index, :show], controller: 'operations'
   end
 
+  # Analytics charts and tables
+  resource :analytics, only: [] do
+    get 'repository_growth', action: :repository_growth
+    get 'repository_object_counts', action: :repository_object_counts
+  end
+
   # Dashboard page
   resource :dashboard, controller: 'dashboard', only: [:show]
   resources :dashboard, only: [] do
     collection do
       get 'activity', action: :activity, as: :dashboard_activity
-      get 'update_collections', action: :update_collections, as: :update_collections
```

Comment:
Does this just disappear?

Reply:
It was changed to use a dataTable and having everything run in the browser. Before you could toggle between several states, that required hitting solr, and hence the update_collections route, which